base/utils/R/unit_is_parseable.R

```diff
@@ -0,0 +1,17 @@
+##' Check whether a unit string is parseable
+##' 
+##' Function will replace the now-unmaintained `udunits2::ud.is.parseable`
+##' @author Tanishq Jain
+##' 
+##' @param u1 A character string representing a type of units
+##' 
+##' @return TRUE if the units is parseable, FALSE otherwise.
+##' @export
+unit_is_parseable <- function(u1){
+  tryCatch({
+    if(units::as_units(u1))
+      return(TRUE)
+  },
+  error = function(e) FALSE
+  )
+} # unit_is_parseable
```

Comment:
@nanu1605 go ahead and add a newline here. Rest everything looks good and clean to me. Let's wait for other reviewers and once we're good, I think you can go ahead and replace the `udunits2::ud.is.parseable()` function in your previous PRs.